diagnostic.md

```diff
@@ -14,7 +17,8 @@ list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
 <!-- your answer starts here -->
-
+$("li").getAttribute
+$("OL").getChildren
```

Comment:
There are a few ways to do this, here's one:

`$('li')`

You may have overthought it. Also remember if you want to invoke a method you have to have `()` after it!